.circleci/config.yml

```diff
@@ -58,6 +64,21 @@ jobs:
       - run:
           name: build and push docker images
           command: 'cd ~/circle/git/fb-publisher && ./scripts/build_platform_images.sh -p test'
+      - run:
+          name: kubectl configure credentials
+          command: 'kubectl config set-credentials "circleci_test" --token="$KUBE_TOKEN_TEST"'
+      - run:
+          name: kubectl configure context
+          command: 'kubectl config set-context "circleci_test" --cluster="$KUBE_CLUSTER" --user="circleci_test" --namespace="formbuilder-publisher-test"'
+      - run:
+          name: kubectl use circleci context
+          command: 'kubectl config use-context circleci_test'
+      - run:
+          name: apply kubernetes changes to test dev
+          command: './scripts/deploy_platform.sh -p test'
+      - run:
+          name: delete pods in test dev
+          command: './scripts/restart_platform_pods.sh -p test -c circleci_test'
```

Comment:
This can be zapped when rolling updates are in place?

Reply:
yes